lib/fcom.rb

```diff
@@ -2,14 +2,23 @@
 
 # This `Fcom` module is the namespace within which most of the gem's code is written.
 module Fcom
+  ROOT_PATH = '.'
+
   def self.define_slop_options(options)
     git_helpers = Fcom::GitHelpers.new
     default_repo = git_helpers.repo || 'username/repo'
 
     options.string('--repo', 'GitHub repo (in form `username/repo`)', default: default_repo)
     options.integer('-d', '--days', 'number of days to search back')
     options.bool('-r', '--regex', 'interpret search string as a regular expression', default: false)
-    options.bool('-p', '--parse-mode', 'whether we are in parse mode', default: false, help: false)
+    options.string(
+      '-p',
+      '--path',
+      'path (directory or file) used to filter results',
+      default: Fcom::ROOT_PATH,
+    )
+    options.bool('--parse-mode', 'whether we are in parse mode', default: false, help: false)
+    options.bool('--debug', 'activate debug mode', default: false, help: false)
```

Comment:
this new `--debug` option is an unrelated change to make local testing easier while doing development